src/components/Auth/AuthComponent/AuthComponent.jsx

```diff
@@ -0,0 +1,60 @@
+import { observe, streamProps } from "frint-react";
+import React from "react";
+
+import {
+  loginUser,
+  logoutUser,
+  resetPassword,
+  signUpUser,
+} from "../../../actions/Auth/auth";
+import AccountComponent from "../AccountComponents/AccountComponent";
+import LoginComponent from "../LoginComponent";
+
+import "./AuthComponent.css";
+
+const Auth = props => {
+  const [dialogOpen, setDialogOpen] = React.useState(false);
+  const handleAuthDialogOpen = () => {
+    setDialogOpen(true);
+  };
+  const handleAuthDialogClose = () => {
+    setDialogOpen(false);
+  };
+
+  return (
+    <>
+      {props.loggedIn ? (
+        <AccountComponent {...props} unreadCount={4} />
+      ) : (
+        <LoginComponent
+          {...props}
+          handleClose={handleAuthDialogClose}
+          handleAuthDialogOpen={handleAuthDialogOpen}
+          open={dialogOpen}
+        />
+      )}
+    </>
+  );
+};
+
+export default observe(app =>
+  streamProps()
+    .set(app.get("store").getState$(), state => ({
+      loggedIn: state.authReducer.loggedIn,
```

Comment:
The object `{ loggedIn: ... }` could be extracted (to a function in the same file).